FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -373,30 +455,74 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     </msContents>
                         <physDesc>
                             <objectDesc>
-                                <layoutDesc><layout columns="2" writtenLines="25"></layout></layoutDesc><!-- copied from part 2 - or different? -->
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="21" corresp="p3">
+                                        <locus from="122rb" to="122vb"/>
+                                        <note corresp="#textarea5">Data on text area dimensions are taken from <locus target="#122v"/>.</note>
+                                        <dimensions unit="mm" xml:id="textarea5">
+                                            <height>235</height>
+                                            <width>215</width>
+                                        </dimensions>
+                                        <ab type="pricking">Pricking visible</ab>
+                                        <ab type="ruling">Ruling visible</ab>
+                                    </layout>
+                                    <layout columns="2" writtenLines="18" corresp="p3">
```

Comment:
`xml:id `textarea 6 (not necessary, but if you insist you need it) must go to `layout`

Reply:
removed textareas